### PR TITLE
Feature/module effects

### PR DESCRIPTION
**Pull request by yalania**

### CODE REVIEW

**Review by tonifj — APPROVED** [review]
All correct

File: Engine/Module/ModuleRender.cpp [hunk 1]
```diff
@@ -2,22 +2,19 @@
 
 #include "Main/Globals.h"
 #include "Main/Application.h"
+#include "ModuleEffects.h"
```
Comment (raulgonzalezupc) [inline Engine/Module/ModuleRender.cpp]: Alphabetical order

File: Engine/Main/Application.cpp [hunk 2]
```diff
@@ -4,6 +4,7 @@
 #include "Module/ModuleAI.h"
 #include "Module/ModuleAudio.h"
 #include "Module/ModuleAnimation.h"
+#include "Module/ModuleEffects.h"
```
Comment (raulgonzalezupc) [inline Engine/Main/Application.cpp]: after module camera

File: Engine/Module/ModuleEffects.cpp [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	{
+		billboard->Render(billboard->owner->transform.GetGlobalTranslation());
+	}
+	for (auto &particles : particle_systems)
+	{
+		particles->Render();
+	}
+}
+
+ComponentBillboard * ModuleEffects::CreateComponentBillboard()
```
Comment (raulgonzalezupc) [inline Engine/Module/ModuleEffects.cpp]: ComponentBillboard* ModuleEffects::CreateComponentBillboard()

File: Engine/Module/ModuleEffects.cpp [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+{
+	ComponentBillboard* created_billboard = new ComponentBillboard();
+	billboards.push_back(created_billboard);
+	return created_billboard;
+}
+
+void ModuleEffects::RemoveComponentBillboard(ComponentBillboard * billboard_to_remove)
```
Comment (raulgonzalezupc) [inline Engine/Module/ModuleEffects.cpp]: oid ModuleEffects::RemoveComponentBillboard(ComponentBillboard* billboard_to_remove)

File: Engine/Module/ModuleEffects.cpp [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+{
+	auto it = std::find(billboards.begin(), billboards.end(), billboard_to_remove);
+	if (it != billboards.end())
+	{
+		delete *it;
+		billboards.erase(it);
+	}
+}
+
+ComponentParticleSystem * ModuleEffects::CreateComponentParticleSystem()
```
Comment (raulgonzalezupc) [inline Engine/Module/ModuleEffects.cpp]: same

File: Engine/Module/ModuleEffects.cpp [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+{
+	ComponentParticleSystem* created_particle_system = new ComponentParticleSystem();
+	particle_systems.push_back(created_particle_system);
+	return created_particle_system;
+}
+
+void ModuleEffects::RemoveComponentParticleSystem(ComponentParticleSystem * particle_system_to_remove)
```
Comment (raulgonzalezupc) [inline Engine/Module/ModuleEffects.cpp]: same

**Review by raulgonzalezupc — APPROVED** [review]
Nice!

**Pull request merged**